admin/admin.less

```diff
@@ -307,10 +307,211 @@
 		font-style: italic;
 	}
 
+	#sow-settings-dialog {
```

Comment:
This is the problem with having all our plugins that work together but need to function independently. Not sure on a good alternative, but at least these won't both be enqueued on the same page. This `admin.less` is only used on the Widgets Bundle admin page _Plugins > SiteOrigin Widgets_